src/hotspot/share/runtime/arguments.cpp

```diff
@@ -537,42 +536,9 @@ static SpecialFlag const special_jvm_flags[] = {
   { "TLABStats",                    JDK_Version::jdk(12), JDK_Version::undefined(), JDK_Version::undefined() },
 
   // -------------- Obsolete Flags - sorted by expired_in --------------
-  { "PermSize",                      JDK_Version::undefined(), JDK_Version::jdk(8),  JDK_Version::undefined() },
```

Comment:
No objecting, but it doesn't look like we told them when it would become unrecognized.

Reply:
We don't report that value in the warning we only report the release in which it became obsolete. e.g.:

Java HotSpot(TM) 64-Bit Server VM warning: Ignoring option PermSize; support was removed in 8.0

So to all intents and purposes you can read all these undefined() entries as if they said jdk(17).

Thanks for the review.